js/src/dapps/registry/Events/events.js

```diff
@@ -122,7 +183,18 @@ export default class Events extends Component {
 
         return evB.timestamp - evA.timestamp;
       })
-      .map((e) => eventTypes[e.type](e, accounts, contacts));
+      .map((e) => eventTypes[e.type](e));
+
+    const reverseToggled = (
```

Comment:
Yup: Prefer to wrap multiline conditions in parenthesises. Less error-prone and more readable.